src/modules/cacheClient/ICacheClient.ts

```diff
@@ -11,9 +11,9 @@ import {
     IOrganization,
     IRole,
     Order,
-} from "./cacheServerClient.types";
+} from "./cacheClient.types";
 
-export interface ICacheServerClient {
+export interface ICacheClient {
```

Comment:
Maybe not necessary for the initial refactoring, but I think it might be cool to refactor this interface as well to be by domain. E.g. `IAssetCacheClient`, `IDomainCacheClient`, `IDIDCacheClient`